tests/components/device_tracker/test_init.py

```diff
@@ -12,20 +12,25 @@
     ATTR_ENTITY_ID, ATTR_ENTITY_PICTURE, ATTR_FRIENDLY_NAME, ATTR_HIDDEN,
     STATE_HOME, STATE_NOT_HOME, CONF_PLATFORM)
 import homeassistant.components.device_tracker as device_tracker
+from homeassistant.exceptions import HomeAssistantError
 
 from tests.common import (
     get_test_home_assistant, fire_time_changed, fire_service_discovered)
 
+TEST_PLATFORM = {device_tracker.DOMAIN: {CONF_PLATFORM: 'test'}}
+
 
 class TestComponentsDeviceTracker(unittest.TestCase):
     """Test the Device tracker."""
+    hass = None  # HomeAssistant
+    yaml_devices = None  # type: str
 
-    def setUp(self):  # pylint: disable=invalid-name
+    def setup_method(self, _):
```

Comment:
Based on trying to get rid of pylint disables, yes.

But also not wise to follow rules blindly. Keep the disable?